sapi/fpm/fpm/fpm_sockets.c

```diff
@@ -526,6 +526,32 @@ int fpm_socket_get_listening_queue(int sock, unsigned *cur_lq, unsigned *max_lq)
 
 #endif
 
+#ifdef HAVE_LQ_TCP_CONNECTION_INFO
```

Comment:
might be nicer to do `#elif defined(HAVE_LQ_TCP_CONNECTION_INFO)` so it's clear that it can be defined only once.